commit 37c815ee660d1bf37256638d23b0346ad7cc19e7
Author: sternenseemann <0rpkxez4ksa01gb3typccl0i@systemli.org>
Date:   Wed Jul 21 00:18:30 2021 +0200

    omp/setup.py: open version_information in text mode
    
    Otherwise saving the config in setup_support.py will fail as a bytes
    object is not encodeable as JSON. Luckily, version_information is text
    anyways.

diff --git a/omp/setup.py b/omp/setup.py
index 942ab1f5..5281398e 100755
--- a/omp/setup.py
+++ b/omp/setup.py
@@ -25,7 +25,7 @@ class GNATCollOMP(SetupApp):
 
         # Set library version
         with open(os.path.join(config.source_dir, '..',
-                               'version_information'), 'rb') as fd:
+                               'version_information'), 'r') as fd:
             version = fd.read().strip()
         config.set_data('GNATCOLL_VERSION', version, sub='gprbuild')
 
